wbooks/src/App.js

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
-import { Switch, Route } from 'react-router-dom'
 import './App.css';
-import BooksDashboard from './app/screens/BooksDashboard';
-import BookDetail from './app/screens/BookDetail/BookDetail.js';
+import BooksDashboard from './app/screens/BooksDashboard.js';
+import BookDetail from './app/screens/BookDetail/BookDetail.js'
+import Login from './app/screens/Login/Login.js'
+import {BrowserRouter} from 'react-router-dom'
+import { Switch, Route } from 'react-router-dom'
+import AuthRoute from './app/components/authRoute/AuthRoute.js'
 
 class App extends React.Component {
+
   render() {
+
+  const accessToken = localStorage.getItem('access_token')
+  console.log(accessToken);
     return (
       <Switch>
-        <Route exact path='/' component={BooksDashboard}/>
-        <Route exact path='/book/:id' component={BookDetail}/>
+        <AuthRoute exact path="/" component={BooksDashboard} isPrivate accessToken={accessToken} />
```

Comment:
Instead of passing down `accessToken={accessToken}`, get the token inside `AuthRoute`.